src/lm.jl

```diff
@@ -119,11 +122,20 @@ const FIT_LM_DOC = """
     is less-than-full rank. If `true` (the default), only the first of each set of
     linearly-dependent columns is used. The coefficient for redundant linearly dependent columns is
     `0.0` and all associated statistics are set to `NaN`.
+
+    `contrasts` may optionally be supplied in the form of a `Dict` mapping term names
+    (as `Symbol`s) to term types (e.g. `ContinuousTerm`) or contrasts
+    (e.g., `HelmertCoding()`, `SeqDiffCoding(; levels=["a", "b", "c"])`,
+    etc.). If contrasts are not provided for a variable, the appropriate
+    term type will be guessed based on the data type from the data column:
+    any numeric data is assumed to be continuous, and any non-numeric data
+    is assumed to be categorical.
```

Comment:
should this be factored out to `const CONTRASTS_DESC` to make it easy to e.g. address my comment above everywhere?